### PR TITLE
Update slovak.xml

### PR DESCRIPTION
**Pull request by kubalav**

### CODE REVIEW

**Comment by donho** [comment]
@kubalav 
You opened also this PR:
https://github.com/notepad-plus-plus/notepad-plus-plus/pull/8636
Which one should I take?

**Comment by kubalav** [comment]
This one is the latest. I closed previous PR.

**Comment by donho** [comment]
So did you check the following link to make sure your PR include everything ?
http://download.notepad-plus-plus.org/repository/MISC/diff/english.diff.htm

**Comment by donho** [comment]
@kubalav  Please let me know when your update is done.

**Comment by kubalav** [comment]
Done.

File: PowerEditor/installer/nativeLang/slovak.xml [hunk 1]
```diff
@@ -978,17 +1052,17 @@ Zadať môžete viacero značiek pomocou medzery na oddelenie rôznych čísel."
...
-                <Item id="2025" name="Oct"/>
-                <Item id="2026" name="Hex"/>
-                <Item id="2027" name="Bin"/>
+                <Item id="2023" name="Vložiť &amp;text"/>
+                <Item id="2033" name="Vložiť čí&amp;slo"/>
+                <Item id="2030" name="&amp;Začiatočné číslo:"/>
+                <Item id="2031" name="Z&amp;vyšovať po:"/>
+                <Item id="2035" name="&amp;Nuly na začiatku"/>
+                <Item id="2036" name="&amp;Počet opakovaní:"/>
+                <Item id="2032" name="&amp;Formát"/>
```
Comment (donho) [inline PowerEditor/installer/nativeLang/slovak.xml:1061]: Sorry I just fixed an issue regarding it: https://github.com/notepad-plus-plus/notepad-plus-plus/commit/55d671719c8f00f02ed32b15575bc2e6587a5e30
`&amp;` in front of 'F' should be removed.

**Review by donho — CHANGES_REQUESTED** [review]
Please remove `&amp;` in front of 'F'.

**Review by donho — APPROVED** [review]

**Pull request closed without merging**